projects/data-grid-demo/src/app/app.component.ts

```diff
@@ -26,7 +28,16 @@ export class AppComponent implements OnInit {
   constructor(private store: Store<any>) {
     this.config = GridConfigBuilder.gridConfig().withCheckboxSelection(true);
     this.data = new MockService().getData().rows;
+    this.data[0].trafficLights = {
```

Comment:
Let's model be like this:
this.data[0].badges = {
      requester: [{id: 1, status: {id: 'A'}}, {id: 2, status: {id: 'P'}}],
      manager: [{id: 3, status: {id: 'U'}}, {id: 4, status: {id: 'P'}}],
      coordinator: [{id: 5, status: {id: 'A'}}, {id: 6, status: {id: 'R'}}]
    };